src/mobile/src/components/Banner/Banner.jsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
I thought we had a fix before to allow us not having ESlint to show an error when react not imported